tests/system/feature_store/test_feature_store.py

```diff
@@ -4202,6 +4202,31 @@ def test_get_online_features_after_ingest_without_inference(self):
                 f"store://feature-vectors/{self.project_name}/my-vector:latest"
             )
 
+    def test_ingest_with_rename_columns(self):
+        csv_path = str(self.assets_path / "fields_with_space.csv")
+        name = f"test_ingest_with_rename_columns_{uuid.uuid4()}"
+        data = pd.read_csv(csv_path)
+        expected_result = data.copy().rename(columns={"city of birth": "city_of_birth"})
+        expected_result.set_index("name", inplace=True)
+        feature_set = fstore.FeatureSet(
+            name=name,
+            entities=[fstore.Entity("name")],
+        )
+        fstore.preview(
+            feature_set,
+            data,
+        )
+        result = fstore.ingest(
+            feature_set, data, run_config=fstore.RunConfig(local=True)
```

Comment:
no need for specifying RunConfig(local=True) since it's the default.
also for readability, please rename `result` to `inspect_result`